Sources/XiEditor/RPCSending.swift

```diff
@@ -40,6 +40,41 @@ enum RpcResult {
     case ok(AnyObject)
 }
 
+enum RPCRequestMethod: String {
+    case measureWidth = "measure_width"
+}
+
+enum RPCNotificationMethod: String {
+    case alert
+    
+    case update
```

Comment:
is there a reason these two don't have raw values?

Reply:
it will use the case label directly....it would be redundant to have like `case alert = "alert"`

Reply:
Ah cool I didn't know that. 👍